packages/adapter-drizzle/src/pg/index.ts

```diff
@@ -0,0 +1,231 @@
+    providerAccountId: text("providerAccountId").notNull(),
+    refresh_token: text("refresh_token"),
+    access_token: text("access_token"),
+    expires_at: integer("expires_at"),
+    token_type: text("token_type"),
+    scope: text("scope"),
+    id_token: text("id_token"),
+    session_state: text("session_state"),
+  },
+  (account) => ({
+    compoundKey: primaryKey(account.provider, account.providerAccountId),
+  })
+)
+
+/** @internal */
+export const sessions = pgTable("sessions", {
+  sessionToken: text("sessionToken").notNull().primaryKey(),
+  userId: text("userId")
+    .notNull()
+    .references(() => users.id, { onDelete: "cascade" }),
+  expires: timestamp("expires", { mode: "date" }).notNull(),
+})
+
+/** @internal */
+export const verificationTokens = pgTable(
+  "verificationToken",
+  {
+    identifier: text("identifier").notNull(),
+    token: text("token").notNull(),
+    expires: timestamp("expires", { mode: "date" }).notNull(),
+  },
+  (vt) => ({
+    compoundKey: primaryKey(vt.identifier, vt.token),
+  })
+)
+
+/** @internal */
+export const schema = { users, accounts, sessions, verificationTokens }
+export type DefaultSchema = typeof schema
+
+/** @internal */
+export function pgDrizzleAdapter(
+  client: PostgresJsDatabase<Record<string, never>>
+): Adapter {
+  return {
+    createUser: async (data) => {
+      return await client
+        .insert(users)
+        .values({ ...data, id: crypto.randomUUID() })
+        .returning()
+        .then((res) => res[0] ?? null)
+    },
+    getUser: async (data) => {
+      return await client
+        .select()
+        .from(users)
+        .where(eq(users.id, data))
+        .then((res) => res[0] ?? null)
+    },
+    getUserByEmail: async (data) => {
+      return await client
+        .select()
+        .from(users)
+        .where(eq(users.email, data))
+        .then((res) => res[0] ?? null)
+    },
+    createSession: async (data) => {
+      return await client
+        .insert(sessions)
+        .values(data)
+        .returning()
+        .then((res) => res[0])
+    },
+    getSessionAndUser: async (data) => {
+      return await client
+        .select({
+          session: sessions,
+          user: users,
+        })
+        .from(sessions)
+        .where(eq(sessions.sessionToken, data))
+        .innerJoin(users, eq(users.id, sessions.userId))
+        .then((res) => res[0] ?? null)
+    },
+    updateUser: async (data) => {
+      if (!data.id) {
+        throw new Error("No user id.")
+      }
+
+      return await client
+        .update(users)
+        .set(data)
+        .where(eq(users.id, data.id))
+        .returning()
+        .then((res) => res[0])
+    },
+    updateSession: async (data) => {
+      return await client
+        .update(sessions)
+        .set(data)
+        .where(eq(sessions.sessionToken, data.sessionToken))
+        .returning()
+        .then((res) => res[0])
+    },
+    linkAccount: async (rawAccount) => {
```

Comment:
If you get the following error with Github provider:
```
      "message": "Cannot read properties of undefined (reading 'name')",
      "stack": "TypeError: Cannot read properties of undefined (reading 'name')\n
```
It's because GitHub adds an extra property `refresh_token_expires_in`: 
```typescript
{
  provider: 'github',
  type: 'oauth',
  providerAccountId: '',
  access_token: '',
  expires_at: 1690073663,
  refresh_token: '',
  refresh_token_expires_in: 15897600, // <----- 
  token_type: 'bearer',
  scope: '',
  userId: '53c777ed-353a-438b-a57e-wqweqwe'
}
```
I think maybe it would be best (for compatibility) to add this column to the provider? Or I guess maybe point that out in the docs so people can add that column to the schema? 

Also, it may be better to use bigInt (with mode `number`) instead of integer for the expires column(s) for flexibility. https://orm.drizzle.team/docs/column-types/pg#bigint